lib/origen_testers/smartest_based_tester/v93k_smt8/limits_workbook.rb

```diff
@@ -21,6 +21,7 @@ def add_softbin(number, options = {})
             priority: 2
           }.merge(options)
           attrs = @softbins[number] || {}
+          options[:priority] = options[:priority] || 2
```

Comment:
Is this line necessary?

Reply:
'2' kind of redundant here with line 21 also present.